openbas-api/src/main/java/io/openbas/helper/InjectHelper.java

```diff
@@ -53,9 +50,7 @@ private List<Team> getInjectTeams(@NotNull final Inject inject) {
   // -- INJECTION --
 
   private Stream<Tuple2<User, String>> getUsersFromInjection(Injection injection) {
-    if (injection instanceof DryInject dryInject) {
-      return dryInject.getRun().getUsers().stream().map(user -> Tuples.of(user, "Dryrun"));
-    } else if (injection instanceof Inject inject) {
+    if (injection instanceof Inject inject) {
```

Comment:
question: now that we only have one type of inject, couldn't we change the method signature to take directly an Inject instead of an Injection, thus eliminating the instanceof check and the throw of unsupported operation?